judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/problem/bundle/ItemConfig.java

```diff
@@ -1,5 +1,21 @@
 package judgels.sandalphon.api.problem.bundle;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.function.Function;
+
 public interface ItemConfig {
     String getStatement();
+    ItemConfig processDisplayText(Function<String, String> processor);
+    ItemConfig withoutGradingInfo();
```

Comment:
I think the correct name is actually `withoutAnswerKey()`. Because "grading" info should also include e.g. penalty and score, which is not excluded here.

Reply:
Changed